### PR TITLE
New Config & Features

### PR DESCRIPTION
**Pull request by brunofoster03**
Re-configured app and added Flutter routes, application theme configuration and Home Screen

### CODE REVIEW

File: .gitignore
Comment (manduinca) [inline .gitignore:1]: Why do we need to change it?

**Comment by roquemolina** [comment]
We've been working together with this repo and it works. I was neccessary to implement Isaac's views.

File: .vscode/settings.json
Comment (manduinca) [inline .vscode/settings.json:1]: I think we don't need to upload this file.

**Review by roquemolina — DISMISSED** [review]
We've been working together with this repo and it works. I was neccessary to implement Isaac's views.

**Review by manduinca — CHANGES_REQUESTED** [review]
I leaved some comments.

**Review by IsaacMartinez123 — COMMENTED** [review]
Good

**Review by IsaacMartinez123 — APPROVED** [review]
Buenisimo

**Comment by brunofoster03** [comment]
Resolved

File: .gitignore
Comment (roquemolina) [reply .gitignore:1]: resolved

File: .vscode/settings.json
Comment (roquemolina) [reply .vscode/settings.json:1]: resolved

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by roquemolina — APPROVED** [review]
OK!

**Pull request merged**